packages/calling-stateful-client/src/InternalCallContext.ts

```diff
@@ -51,8 +48,8 @@ export class InternalCallContext {
   // <CallId, <ParticipantKey, <StreamId, RemoteRenderInfo>>
   private _remoteRenderInfos = new Map<string, Map<string, Map<number, RemoteRenderInfo>>>();
 
-  // <CallId, LocalRenderInfo>.
-  private _localRenderInfos = new Map<string, LocalRenderInfo>();
+  // <CallId, <MediaStreamType, LocalRenderInfo>>.
```

Comment:
This is the key change in this PR. Instead of having a single LocalRenderInfo per Call, we now have a map of streamType to render info per call.
This supports having one video stream, one screenshare stream, one raw video access stream etc.

There isn't a good streamId to use instead of MediaStreamType to allow multiple local camera streams/multiple local screensharestreams etc (instead of just one of each). We could use stream.source.id but that key gets messy to track when switchSource happens